### DISCUSSION TITLE
Why did not the checks get triggered?

### DISCUSSION THREAD

**Discussion by dongho-jung** [community/community#38162] [Other Feature Feedback, Questions, & Ideas]
At "Nov 3, 2022, 5:11 PM GMT+9", I pushed a commit and it should've triggered some workflows and checks. However, it didn't at all.

No checks or workflows were triggered.. just after that I pushed empty commit to retrigger it, it was triggered as I expected.

I never seen this before, Is there anybody know what happened then?

If you need more information I'll try my best to offer it. But It was really weird because there was no particular things at all..

**Comment by david-engelmann** [comment]
I've experienced delays before. From the actions tab, Is the event still not present?
